src/pages/plugins-2023/index.md.ejs

```diff
@@ -13,6 +13,27 @@ On this page, we provide an `npm` link and total downloads per month for any plu
 we find or publish ourselves, so you only need to look in one place.
 
 <!-- Plugin groups -->
+<!-- Plugin groups v3 -->
+<%
+let v3groups = '<table><tr>'
+let v3 = 1
+Object.keys(allPlugins).forEach(function (group) {%>
+  ### <%=group%>
+  <%Object.keys(allPlugins[group]).forEach(function (pluginName) {
```

Comment:
the ejs syntax can get quite hard to read - unless you are using <%= 
place the escape tags <% and %> on separate lines